topology/platform/common/ssp.m4

```diff
@@ -26,8 +26,8 @@ define(`SSP_CONFIG',
 $6
 )
 
-dnl SSP_CONFIG_DATA(type, idx, valid bits, mclk_id)
-dnl mclk_id is optional
+dnl SSP_CONFIG_DATA(type, idx, valid bits, mclk_id, quirks, frame_pulse_width)
+dnl mclk_id quirks frame_pulse_width are optional
```

Comment:
@xiulipan is working on something similar which will restart after 1.2. Please align.

Reply:
this PR should be align with the kernel's PR "ssp:support additional SSP register bits #64".
the kernel's PR is merged. 
I hope the tplg's update will not take much time.
the incompatible would cause problem.

Reply:
@lgirdwood 
I would like to use some more vendor token for this. Because the alse-tplg have no such more config we have. we need to keep these config as bespoke vendor tuple.